book/src/SUMMARY.md

```diff
@@ -5,7 +5,112 @@
 
 # User Guide
 
+## Multi-threading
```

Comment:
SUMMARY.md is supposed to contain a TOC with links. please move the content to per-chapter md files.

Reply:
Done with 4726e45fe247e5ff9527acbcfa5ee6483e2829ff.